packages/okta-react-native/OktaSdkBridgeReactNative.podspec

```diff
@@ -0,0 +1,20 @@
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name         = package['name']
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.license      = package['license']
+
+  s.authors      = package['author']
+  s.homepage     = package['homepage']
+  s.platform     = :ios, "9.0"
+
+  s.source       = { :git => "https://github.com/FormidableLabs/react-native-app-auth.git" }
```

Comment:
FormidableLabs? :)  Please change it to Okta link

Reply:
Whoops :D